core/src/main/java/io/parsingdata/metal/Util.java

```diff
@@ -61,17 +62,10 @@ public static boolean notNullAndSameClass(final Object object, final Object othe
             && object.getClass() == other.getClass();
     }
 
-    public static String bytesToShortHexString(final byte[] bytes) {
+    public static String bytesToHexString(final byte[] bytes) {
```

Comment:
Why is the max to limit the print removed here?

Reply:
Ah, I have seen the max in value, which limits the size when he is called.

Reply:
Correct. The other place where this is called is from `ConstantSource`, I didn't limit the size there because I figured if you use a constant it will be (a) not very big and (b) it's desirable to see it fully printed.